### ISSUE TITLE
getTagWeight not being called

### ISSUE THREAD

**Issue by colinbes** [rantecki/docpad-plugin-tagcloud#4]
I am not sure if I am doing something wrong but it doesn't appear as if getTagWeight is being callled when I override the function in docpad.coffee. 

Irrespective of what I override, tag data doesn't change and I assume the default is always running.

I have tried using plugin name 'tagging' and 'tagcloud', code below

```
plugins:
   tagging:
        getTagWeight: (count, maxCount) ->
            return 1+(count/maxCount)
```

and using tagcloud - just in case..

```
plugins:
    tagcloud:
        getTagWeight: (count, maxCount) ->
            return 1+(count/maxCount)
```

**Comment by rantecki** [comment]
What version of docpad are you running?  Perhaps they've changed something again...

**Comment by colinbes** [comment]
I am running v6.69.0 on osx mavericks. More than happy to run any tests if you want. This is my first foray into docpad so I wouldn't rule user error out but it seems that it should be simple enough to setup. 
Can you confirm that plugin name is 'tagging'

**Comment by rantecki** [comment]
Sorry, the plugin name is 'tagcloud'.  'tagging' was it's predecessor and I obviously hadn't updated the docs well enough!

Try running docpad with the `-d` flag to turn on debugging (i.e. `docpad -d generate`) and then post the output.

**Comment by colinbes** [comment]
Wow, it turns out that the problem was that in one of my lines I had couple of spaces instead of only tabs. The compiler process didn't complain until I started adding console.log() statements to debug. With only tabs the override function works. Interesting thing is that (and I confirmed it again) is that no errors are thrown without the console.log statement. Apologies for the false alarm, but at least we got to confirm the plugin name.
